### ISSUE TITLE
Final Presentation information

### ISSUE THREAD

**Issue by ZakBrinlee** [ZakBrinlee/Project-8#48]
Hello Team, 

Please provide any information you have for the following information for the final presentation. 

- Difference between original plan & final delivery

- Key challenges you overcame

- Key unresolved issues or unfinished features

- What you learned through the project

**Comment by TyReed12** [comment]
**Changes**
I was originally assigned the messaging functionality of the app. After quite a bit of help from Zak we have working two way messaging via SendBird. However, we were unable to add photo and media functionality to messaging.

**Key Challenges**
I really struggled with implementing messenger tutorials that worked with React-Native and Expo. I learned that at my current skill level in JavaScript, I really need to go through the tutorial as a brand new app in order to understand each step and how it works before trying to implement it into our constantly shifting code. If I would of done that, I feel that I would of been able to get working messenger functionality much faster. That’s eventually what I did using chatkit and it worked so much better.

**Key unresolved issues or unfinished features**
The messenger still is unable to send photos or other media. It also could be styled much cleaner. Ideally I would like it to look like an FB messenger.

**What you learned through the project**
I learned that my JavaScript skills were pretty poor and needed a lot of work. But I really enjoyed learning how to build mobile apps using React-Native and I don’t think I’ll ever build use native to build a personal app. I also learned that having a competent project lead is huge. Zak put out so many fires in this project and was a huge help in everything I did.

**Comment by bradleyshao** [comment]
Key challenges: I don't have much experience about React native (Javascript). So i need to do lots of research during the coding,and made some stupid mistakes again and again, which makes me wasted lots of time. The biggest challenge is the image upload part. I spent lots of time to figure out how to add permission. But still can't figure how to fix the 'unexpected url' error when i trying to upload image uri to firebase.

Key unresolved issues or unfinished features: The unresolved issue is the uploading uri part. I followed a tutorial on youtube about uploading image uri to firebase. However, it don't works on this project, i keep getting the 'unexpected url' error, and still can't fix it.   

What you learned through the project: I need to improve my Javascript skill. But I really enjoyed the coding process and learned a lot during the research. Zak and Jon Adler gave me lots of help this quarter.
